opal/mca/common/ucx/common_ucx.c

```diff
@@ -83,8 +83,8 @@ OPAL_DECLSPEC void opal_common_ucx_mca_var_register(const mca_base_component_t *
       opal_common_ucx.tls = default_tls;
     }
 
-    tls_index = mca_base_var_register(
-        "opal", "opal_common", "ucx", "tls",
+    tls_index = mca_base_component_var_register(
```

Comment:
why is it needed only for these vars and not for other vars (such as progress_index/hook_index for example)?

Reply:
Only the string variables causing segfaults